api/src/server.ts

```diff
@@ -11,25 +13,44 @@ import {
   PORT
 } from './utils/env';
 
+const requestSerializer = (request: FastifyRequest) => ({
+  method: request.method,
+  url: request.url,
+  ip: request.headers['x-forwarded-for']?.toString() || request.ip,
```

Comment:
```suggestion
  ip: request.headers['x-forwarded-for'] || request.ip,
```
It's `undefined` or a string, so if it's truthy we can use it as is.